.github/workflows/deploy-treasury.yml

```diff
@@ -65,35 +65,35 @@ jobs:
 #    outputs:
 #      deployments: ${{ steps.set-deployments.outputs.deployments }}
 
-  deploy:
```

Comment:
I'd prefer renaming `deploy-treasury.yml` to `deploy-amm.yml.DISABLED`.

Reply:
Done